### DISCUSSION TITLE
Not able to access copilot on my education liscence

### DISCUSSION THREAD

**Discussion by acmahc** [community/community#145886] [GitHub Education]
### Hi everyone,

My Education Github licence was approved a month ago but i'm still unable to access to Copilot for free. The image shows my options when i click on the tab "Your copilot"

Thank you for your help !

<img width="845" alt="Screenshot 2024-11-29 at 2 29 21 PM" src="https://github.com/user-attachments/assets/a6813668-9556-4076-b78b-e736e32981bd">

**Comment by davevad93** [comment]
Hi @acmahc 

I assume the status of your application is "`Congrats! Your request for @acmahc was approved and your benefits are now available.`", try redeeming your offer by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
